osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -377,6 +346,24 @@ public bool OnPressed(GlobalAction action)
                     holdingForHUD = true;
                     updateVisibility();
                     return true;
+
+                case GlobalAction.ToggleInGameInterface:
+                    switch (configVisibilityMode.Value)
+                    {
+                        case HUDVisibilityMode.Never:
+                            configVisibilityMode.Value = HUDVisibilityMode.HideDuringGameplay;
+                            break;
+
+                        case HUDVisibilityMode.HideDuringGameplay:
+                            configVisibilityMode.Value = HUDVisibilityMode.Always;
+                            break;
+
+                        case HUDVisibilityMode.Always:
+                            configVisibilityMode.Value = HUDVisibilityMode.Never;
+                            break;
+                    }
+                    updateVisibility();
```

Comment:
this is automatically called via bindable flow and didn't need to be added here.